backend/main.py

```
@@ -1,54 +1,343 @@
-from .core.followup_service import FollowUpService
-from .core.query_router import QueryRouter
-from .core.response_service import ResponseService
 
-project_root = Path(__file__).resolve().parent.parent
+# Import config - try different import paths
+def _import_app_config() -> Any:
+    """Import AppConfig from available module paths."""
+    for module_path in ["backend.core.config", ".core.config", "core.config"]:
+        try:
+            module = importlib.import_module(module_path)
+            return getattr(module, "AppConfig")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)